apps/token-manager/app/package.json

```
@@ -41,13 +41,14 @@
     "prettier": "^1.8.2"
   },
   "scripts": {
-    "test": "jest",
     "lint": "eslint ./src",
-    "start": "npm run sync-assets && npm run watch:script & parcel serve index.html -p 3003 --out-dir build/",
-    "build": "npm run sync-assets && npm run build:script && parcel build index.html --out-dir build/ --public-url \".\"",
-    "build:script": "parcel build src/script.js --out-dir build/",
-    "watch:script": "parcel watch src/script.js --out-dir build/ --no-hmr",
-    "sync-assets": "copy-aragon-ui-assets -n aragon-ui ./build && rsync -rtu ./public/ ./build",
+    "start": "npm run sync-assets && npm run watch & parcel serve index.html -p 3003 --out-dir ../dist/",
+    "watch": "parcel watch src/script.js --out-dir ../dist/ --no-hmr",
+    "serve": "parcel serve index.html --out-dir ../dist/ --no-cache",
```

```suggestion
    "serve": "parcel serve index.html --out-dir ../dist/",
```

Let's see how this goes... I think it should be fine.